src/create_character/create_character.cpp

```diff
@@ -23,19 +24,12 @@ bool CheckAndOverwriteSaveFile()
 		utils::Print({"A save file with an already-created character has been "
 					  "detected."});
 		utils::Print({"Do you want to continue with the character creation and "
-					  "overwrite it? (yes/no)"});
+					  "overwrite it?"});
 
-		std::string overwrite_character = "yes";
-		utils::GetUserInput(overwrite_character);
+		std::vector<std::string> const yes_or_no{"Yes", "No"};
+		int overwrite_character = utils::GetUserConstrainedChoice(yes_or_no, true);
```

Comment:
It works only with Typing "Yes" or "No" and not with the number, is this by design, if it's not i think it's a good thing so the player can't make a mistake so easy